docs/src/Flyout.doc.js

```diff
@@ -46,6 +46,13 @@ card(
         description:
           'Depicts if the Flyout shares a relative root with the anchor element',
       },
+      {
+        name: 'role',
+        type: `"alertdialog" | "dialog"`,
```

Comment:
I don't think this is the right way to make this change. It jams in a visual change with a accessibility feature. Perhaps adding `color` (or similar) instead?

Reply:
I changed it to `color` let me know what you think. I originally chose role because it matches `Modal` and because it allows a finite set of types with specific purposes. By changing it to `color` at least to me it kind of dilutes the purpose of each color and it can lead to people alter asking why "green" "blue" "red" etc are not supported. In my eyes its harder to map "orange" to being the old `ErrorFlyout` component, than it is to map a role of `alertdialog` to that. Maybe there is some middle ground here with a `type` prop instead? For example look at the `shouldFocus` prop, ErrorFlyout and Flyout set that differently so it feels weird setting that now based on the color. Thoughts @chrislloyd?